websites/A/asset.party/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "521762564062052393",
+		"name": "Dwifte#0661"
+	},
+	"service": "asset.party",
+	"description": {
+		"en": "Presence showing your activity on SBOX"
+	},
+	"url": "asset.party",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/vWSslVh.png",
```

Comment:
Logos should be 512x512px at minimum. Find or create a larger version (using something like waifu2x).